src/createServer.js

```diff
@@ -1,10 +1,37 @@
-'use strict';
+const http = require('http');
+const { sendTextResponse } = require('./sendResponse');
+const { readFileByPath } = require('./getFilePath');
+const { getError } = require('./error');
+const url = require('url');
 
-function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
-}
+const createServer = () => {
+  const server = http.createServer(async (req, res) => {
+    try {
+      const normalizedUrl = new url.URL(
+        req.url || '',
+        `http://${req.headers.host}`,
+      );
 
-module.exports = {
-  createServer,
+      const requestedPath = normalizedUrl.pathname.slice(1);
+      const error = getError(requestedPath);
```

Comment:
The `getError` function should validate that the `requestedPath` starts with `/file/`, as per the task requirements. Ensure this validation is implemented correctly.